doc/rados/troubleshooting/troubleshooting-pg.rst

```diff
@@ -1,120 +1,128 @@
-=====================
+====================
  Troubleshooting PGs
-=====================
+====================
 
 Placement Groups Never Get Clean
 ================================
 
-When you create a cluster and your cluster remains in ``active``,
-``active+remapped`` or ``active+degraded`` status and never achieves an
-``active+clean`` status, you likely have a problem with your configuration.
+If, after you have created your cluster, it remains in `the `active`` status,
+the ``active+remapped`` status or the ``active+degraded`` status and never
+achieves an ``active+clean`` status, you likely have a problem with your
+configuration.
 
-You may need to review settings in the `Pool, PG and CRUSH Config Reference`_
-and make appropriate adjustments.
+In such a situation ,it may be necessary to review settings in the `Pool, PG
+and CRUSH Config Reference`_ and make appropriate adjustments.
 
-As a general rule, you should run your cluster with more than one OSD and a
-pool size greater than 1 object replica.
+As a general rule, run your cluster with more than one OSD and a pool size
+greater than one object replica.
```

Comment:
Let's advise greater than two.  Some people -- including people who should know better -- think 2R is just dandy.

Reply:
Accepted.